lib/domUpdates.js

```diff
@@ -33,8 +26,13 @@ let domUpdates = {
 
   hideCard() {
     document.querySelector('.js-card-popup').style.zIndex = '-1';
+    $('.js-guess-input').val('');
   },
 
+  createDailyDoubleCard(currentClue) {
+    document.querySelector('.js-daily-double-popup').style.zIndex = '2';
+},
+
```

Comment:
Good thinking on making this z-index set to 2 so that it shows in front of the regular clue pop up.